ui/partials/perf/graphsctrl.html

```diff
@@ -54,6 +54,7 @@
   <div id="graph-tooltip" ng-class="{locked: selectedDataPoint}">
     <div class="body">
       <div>
+        <span id='close-popup'>X</span>
```

Comment:
@wlach I tried using that locally, it is black in color and does not appear distinctively on the black popup box. Do you want me to create a new button to suit our needs?